Code/BasicFilters/json/SLICImageFilter.json

```diff
@@ -0,0 +1,145 @@
+      "inputs" : [
+        "Input/VM1111Shrink-RGB.png"
+      ],
+      "md5hash" : "685aeafe583dd0b8a0471be4c005df02",
+      "measurements_results" : [
+        {
+          "name" : "AverageResidual",
+          "value" : "0.10256",
+          "tolerance" : 0.0001
+        }
+      ]
+    },
+    {
+      "tag" : "minimum",
+      "description" : "Do minimal amount of work, 1 iteration, no connectivity",
+      "settings" : [
+        {
+          "parameter" : "MaximumNumberOfIterations",
+          "value" : 1,
+          "cxx_value" : "1u"
+        },
+        {
+          "parameter" : "EnforceConnectivity",
+          "value" : "false",
+          "python_value" : "False",
+          "R_value" : "FALSE"
+        }
+      ],
+      "inputs" : [
+        "Input/VM1111Shrink-RGB.png"
+      ],
+      "md5hash" : "11de1c8d69340e9acf37f382c1599ff1",
+      "measurements_results" : [
+        {
+          "name" : "AverageResidual",
+          "value" : "0.74575",
+          "tolerance" : 0.0001
+        }
+      ]
+    },
+    {
+      "tag" : "gray_scale",
+      "description" : "running on scalar image",
+      "settings" : [
+        {
+          "parameter" : "SuperGridSize",
+          "type" : "unsigned int",
+          "dim_vec" : 1,
+          "value" : [
+            10,
+            10
+          ]
+        }
+      ],
+      "md5hash" : "c7dc411a0b8cd9b36ca63cb70352369a",
+      "inputs" : [
+        "Input/cthead1.png"
+      ],
+      "measurements_results" : [
+        {
+          "name" : "AverageResidual",
+          "value" : "0.03121",
+          "tolerance" : 0.0001
+        }
+      ]
+    }
+  ],
+  "itk_module" : "SimpleITKFilters",
+  "detaileddescription" : "The Simple Linear Iterative Clustering (SLIC) algorithm groups pixels into a set of labeled regions or super-pixels. Super-pixels follow natural image boundaries, are compact, and are nearly uniform regions which can be used as a larger primitive for more efficient computation. The SLIC algorithm can be viewed as a spatially constrained iterative k-means method.\n\nThe original algorithm was designed to cluster on the joint domain of the images index space and it's CIELAB color space. This implementation works with images of arbitrary dimension as well as scalar, single channel, images and most multi-component image types including ITK's arbitrary length VectorImage .\n\nThe distance between a pixel and a cluster is the sum of squares of the difference between their joint range and domains ( index and value ). The computation is done in index space with scales provided by the SpatialProximityWeight parameters.\n\nThe output is a label image with each label representing a superpixel cluster. Every pixel in the output is labeled, and the starting label id is zero.\n\nThis code was contributed in the Insight Journal paper: \"Scalable Simple Linear Iterative Clustering (SSLIC) Using a\nGeneric and Parallel Approach\" by Lowekamp B. C., Chen D. T., Yaniv Z.",
```

Comment:
Reference doesn't include Terry's name at the end (it is included in the ITK doxygen, though the link to the paper is missing there, we should probably add it).